libs/blocks/notification/notification.js

```diff
@@ -118,15 +142,17 @@ function decorateLayout(el) {
   return foreground;
 }
 
-export default function init(el) {
+const init = async (el) => {
```

Comment:
We do use `export default async function` in the codebase, is there anything preventing us from using that pattern here as well?

Reply:
Not at all, I just picked a random block (believe it was [the Editorial Card](https://github.com/adobecom/milo/blob/38e330d70c9dcf657d59f609c958946b41dc49cb/libs/blocks/editorial-card/editorial-card.js#L122)) and went with the pattern I saw there. Turns out I happened to choose one of the few that uses this pattern by chance 😅